sdks/cpp/build_scripts/msvs_2017_x64_release.bat

```diff
@@ -0,0 +1,22 @@
+@echo off
+REM Copyright 2018 Google Inc. All Rights Reserved.
+REM
+REM Licensed under the Apache License, Version 2.0 (the "License");
+REM you may not use this file except in compliance with the License.
+REM You may obtain a copy of the License at
+REM
+REM     http://www.apache.org/licenses/LICENSE-2.0
+REM
+REM Unless required by applicable law or agreed to in writing, software
+REM distributed under the License is distributed on an "AS IS" BASIS,
+REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+REM See the License for the specific language governing permissions and
+REM limitations under the License.
+
+pushd ..
+if not exist ".build" md ".build"
+pushd ".build"
+cmake .. -G "Visual Studio 15 2017 Win64" -Wno-dev
+cmake --build . --config Release

```

Comment:
might want to add `--target install`

Reply:
What do you think about adding something like `CMAKE_INSTALL_PREFIX=./agones_sdk`?
In that case default installation will be in `sdks/cpp/.build/agones_sdk`. I think that users will not be happy with default Program Files installation location.

Reply:
indeed, default install path is probably not ideal.
How about defaulting to `${PROJECT_SOURCE_DIR}/libs` (or something along those lines).